### PR TITLE
Removed code for PRM/Reporting maintenance

### PR DESCRIPTION
**Pull request by galitskyd**
Once we are ready to remove the maintenance splash pages from Reporting and PRM this can be merged.

### CODE REVIEW

File: templates/myreports/reports.html [hunk 1]
```diff
@@ -12,43 +12,7 @@
 {% endblock %}
```
Comment (aspidites) [inline templates/myreports/reports.html]: Wouldn't it be better to keep this code, just in case we have to do something like this in the future?

Comment (galitskyd) [inline templates/myreports/reports.html]: I wouldn't mind keeping something like this in on our apps but this code and implementation leaves a lot to be desired

Comment (aspidites) [inline templates/myreports/reports.html]: Sounds like a ticket to me :P

Comment (galitskyd) [inline templates/myreports/reports.html]: I'll leave it up to @fezick on figuring out if we want a feature like this.

**Pull request merged**